enatega-multivendor-admin/src/components/Restaurant/CreateRestaurant.jsx

```diff
@@ -527,10 +527,11 @@ const CreateRestaurant = props => {
           <Box>
             <Button
               className={globalClasses.button}
-              disabled={loading}
-              onClick={async e => {
+               disabled={loading}
+                onClick={async e => {
                 e.preventDefault()
-                if (onSubmitValidaiton()) {
+                const isValid = onSubmitValidaiton()
```

Comment:
Previous condition and yours have similar functionality. They only differ, in terms of efficiency and readability.

Reply:
But after updating the code the save button is completely working